built/redirect_based_handler.js

```diff
@@ -102,7 +102,7 @@ var RedirectRequestHandler = /** @class */ (function (_super) {
                     .then(function (request) {
                     // check redirect_uri and state
                     var currentUri = "" + _this.locationLike.origin + _this.locationLike.pathname;
-                    var queryParams = _this.utils.parse(_this.locationLike, true /* use hash */);
```

Comment:
We should be making this change to `src/redirect_based_handler.ts` and then running `tsc` to create the built directory

Reply:
oh yeah... forgot about that